CHANGELOG.md

```diff
@@ -3,3 +3,31 @@
 ## 1.0.0 (Unreleased)
 
 Initial release.
+
+## 0.2.3
+
+Add macros for Epoch & Unix timestamp data.
+
+## 0.2.2
+
+Fix queries with missing data
+
+## 0.2.1
+
+Improve tooltip text.
+
+## 0.2.0
+
+Allow to set data source output location.
+
+## 0.1.2
+
+Allow to cancel and stop requests. Bug fixing.
```

Comment:
same nit as mentioned before in https://github.com/grafana/redshift-datasource/pull/97#discussion_r755910983